### DISCUSSION TITLE
Kart and Remote repositories

### DISCUSSION THREAD

**Discussion by Churros396** [koordinates/kart#908] [Q&A] [answered]
Has Karst been applied for team setups, where changes can be pushed to or pulled from a remote repository? The remote repository would be sitting on our own server. Rollbacks, change history and branching would be necessary aswell. Thank you for your time.

**Comment by rcoup** [comment, marked as answer]
Yes, Kart works great locally and remotely. It's on my list to write into the documentation, but a regular git ssh remote works fine with a recent version of git installed on the server. You can fetch changes from and push changes to the remote server: branching, merging, and conflict resolution happen on the client side.

Koordinates is working on repository hosting support and we'd be happy to get you up & running as an early user, or offer some advice: drop an email to support@koordinates.com. Particularly if you're using larger repositories (GB+).

**Reply by Churros396** [reply]
Thank you
